imbi/templates/openapi.yaml

```diff
@@ -7672,6 +7702,13 @@ paths:
         required: true
         schema:
           type: number
+      - name: includes
+        in: query
+        description: Optionally expand the dependent project
+        schema:
+          type: string
+          enum:
+            - dependency
```

Comment:
By default multiple query parameters are _exploded_ (ref: [OpenAPI 3.1](https://spec.openapis.org/oas/latest.html#fixed-fields-9)) so when we have more than one option, the query would look like `?includes=dependency&includes=something-else`. This could be YAGNI, but we should either:

1. set the `explode` property on the parameter to `false`... or
2. rename the parameter to `include`

If you set `explode: false`, then the parameter value is expected to be comma-separated. FWIW, I also checked the spec for 3.0.x and it matches the 3.1.0 content.

Reply:
Why not just allow it? This instance only has one possible value, but others could have multiple, and then we could continue to use the naming schema of "includes" as a query param. I already have the python backend doing a `self.get_query_arguments` check so if there are other erroneous includes values they're just thrown away.

Reply:
Primarily because it is invalid to use `?includes=one,two,three` in the future due to the API specification. You have to use `?includes=one&includes=two&includes=three` which seems a little strange to me. That and the similar JSON API parameter is named `include` ;)

Reply:
ek I named it wrong, it should be `include`.